components/ErrorBanner/OldDesignatedVotingAccountWarningBanner.tsx

```diff
@@ -13,13 +13,9 @@ import {
 } from "./styles";
 
 export function OldDesignatedVotingAccountWarningBanner() {
-  const {
-    oldDesignatedVotingAccount: {
-      isOldDesignatedVotingAccount,
-      message,
-      designatedVotingContract,
-    },
-  } = useStakingContext();
+  const { oldDesignatedVotingAccount } = useStakingContext();
```

Comment:
most of these changes are the same. We're no longer able to destructure these values directly because we can't destructure undefined. So instead we just get the value and destructure either it or an empty object. This has the effect of making the destructured values undefined if the value from the hook is undefined.